test_suite/tests/expand/derive/enum/de_enum.rs

```diff
@@ -0,0 +1,13 @@
+use serde::Deserialize;
+
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
```

Comment:
We should not be testing the compiler's Debug and PartialEq derives here. I don't want broken tests when the compiler changes those...

Reply:
Thanks, I forgot to remove these.